### PR TITLE
Feature/ch122158/tilesets tabs list

### PR DESCRIPTION
**Pull request by Josmorsot**

### CODE REVIEW

File: lib/assets/javascripts/new-dashboard/components/QuickActions/TIlesetQuickActions.vue [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+            event: 'changePrivacy',
+            shouldBeHidden: this.isSubscription
+          }
+        ]
+      };
+    },
+    actionMode () {
+      return 'mine';
+    },
+    isSubscription () {
```
Comment (padawannn) [inline lib/assets/javascripts/new-dashboard/components/QuickActions/TIlesetQuickActions.vue:45]: I think that we can remove this method

**Review by padawannn — CHANGES_REQUESTED** [review]

File: lib/assets/javascripts/new-dashboard/components/QuickActions/TIlesetQuickActions.vue [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+      );
+    },
+    showModal (componentDefinition, componentPropsData) {
+      this.$modal.show(
+        componentDefinition,
+        componentPropsData,
+        { width: '100%', height: '100%' }
+      );
+    },
+    openQuickactions () {
```
Comment (VictorVelarde) [inline lib/assets/javascripts/new-dashboard/components/QuickActions/TIlesetQuickActions.vue:74]: How about naming like openQuickActions ?

File: lib/assets/javascripts/new-dashboard/components/QuickActions/TIlesetQuickActions.vue [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+    }
+  },
+  computed: {
+    actions () {
+      return {
+        mine: [
+          {
+            name: this.$t('QuickActions.changePrivacy'),
+            event: 'changePrivacy',
+            shouldBeHidden: this.isSubscription
```
[the remark this answers is not in the document]
Comment (Josmorsot) [reply lib/assets/javascripts/new-dashboard/components/QuickActions/TIlesetQuickActions.vue:37]: QuickActions is not finished. It's only the copy of DatasetQuick actions but with only one option. Please, ignore it for know

File: lib/assets/javascripts/new-dashboard/components/QuickActions/TIlesetQuickActions.vue (hunk 1 above)
Comment (Josmorsot) [reply lib/assets/javascripts/new-dashboard/components/QuickActions/TIlesetQuickActions.vue:45]: QuickActions is not finished. It's only the copy of DatasetQuick actions but with only one option. Please, ignore it for know

File: lib/assets/javascripts/new-dashboard/components/QuickActions/TIlesetQuickActions.vue (hunk 2 above)
Comment (Josmorsot) [reply lib/assets/javascripts/new-dashboard/components/QuickActions/TIlesetQuickActions.vue:74]: Sure! I think that is a typo, also in file name. I'll fix it in the next PR

**Review by padawannn — APPROVED** [review]

**Pull request merged**